src/pages/Quiz/QuizScreen/QuizScreen.jsx

```diff
@@ -59,7 +59,7 @@ const QuizScreen = () => {
       </div>
 
       <div className="catConatiner">
-        <img id="cat" src={cat} />
+        <img className="cat" src={cat} />
```

Comment:
Keep this as "id" instead of className because it will affect other cat svg from other component.